src/web/mjs/connectors/Agitoon.mjs

```diff
@@ -0,0 +1,129 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class Agitoon extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'agitoon';
+        super.label = 'Agitoon';
+        this.tags = [ 'webtoon', 'korean' ];
+        this.url = 'https://agit185.com';
+        this.cdns = [];
+        this.img_per = [];
+
+    }
+
+    async _initializeConnector() {
+        await this._getNeededValues();
+    }
+
+    async _getNeededValues() {
+        //https://agit185.com/data/azitoon.js?v=0.42317244219852657
+        //contains
+
+        /*
+        var img_domain = "https://blacktoonimg.com/";
+        var img_domain2 = "https://img.blacktoonimg.com/";
+        var img_domain3 = "https://image.blacktoonimg.com/";
+        var img_domain4 = "https://image2.blackmanaimg.com/";
+        var img_domain5 = "https://image3.blackmanaimg.com/";
+        var img_per = "5";
+        var img_per2 = "20";
+        var img_per3 = "5";
+        var img_per4 = "35";
+        var img_per5 = "35";
+        */
+
```

Comment:
suggestion to clear `this.cdns ` and `this.img_per` before filling with current values.